packages/mobile/src/components/Avatar.tsx

```diff
@@ -52,7 +52,7 @@ export function Avatar(props: Props) {
       {...props}
       defaultCountryCode={defaultCountryCode}
       name={getDisplayName(props)}
-      e164Number={e164Number}
+      e164Number={e164Number || (recipient && recipient.e164PhoneNumber)}
```

Comment:
I see we are using `e164Number || (recipient && recipient.e164PhoneNumber)` twice here. Would it be better to isolate this logic in the separate helper function?